source/dotnet/Library/AdaptiveCards/AdaptivePadding.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using Newtonsoft.Json;
+using System;
+using System.ComponentModel;
+using System.Xml.Serialization;
+
+namespace AdaptiveCards
+{
+    /// <summary>
+    /// Represents padding object.
+    /// </summary>
+    public class AdaptivePadding
+    {
+		/// <summary>
+        /// Initializes <see cref="AdaptivePadding"/>.
+        /// </summary>
+        public AdaptivePadding(AdaptiveSpacing left, AdaptiveSpacing top, AdaptiveSpacing right, AdaptiveSpacing bottom)
+        {
+			Left = left;
```

Comment:
is it expected to have different paddings for all directions? I think later implementation versions only allow a single value